translate/cloud-client/translate_v3_create_glossary.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("LongRunningPromise",  "translate_v3_create_glossary")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-translate
+
+# sample-metadata
+#   title: Create Glossary
+#   description: Create Glossary
+#   usage: python3 translate_v3_create_glossary.py [--project "[Google Cloud Project ID]"] [--project_2 "[Your Google Cloud Project ID]"] [--glossary_id "my_glossary_id_123"]
+
+# [START translate_v3_create_glossary]
+from google.cloud import translate_v3 as translate
+
+def sample_create_glossary(project_id, source_lang_code, target_lang_code, input_uri, glossary_id):
+    """
+    Create Glossary
+
+    Args:
+      glossary_id USEFUL DESCRIPTION
+      This needs to inform the developer what value
+      they should provide to glossary_id.
+      It might even want to make a distinction between
+      the full `name` and the `glossary-id`
+    """
+
+    client = translate.TranslationServiceClient()
+
+    # TODO(developer): Uncomment and set the following variables
+    # project_id = 'YOUR_PROJECT_ID'
+    # glossary_id = 'glossary-id'
+    location = 'us-central1'  # The location of the glossary
+
+    name = client.glossary_path(
+        project_id,
+        location,
+        glossary_id)
+    language_codes_set = translate.types.Glossary.LanguageCodesSet(
+        language_codes=[source_lang_code, target_lang_code])
```

Comment:
Since this is language code set, using `source` and `target` is somewhat misleading.

Reply:
I would almost say, don't get them variable names, just set them directly here. 

So:
```
language_codes_set = translate.types.Glossary.LanguageCodesSet(
        language_codes=['en', 'ja'])
```

Reply:
Do we want to make these changes throughout all features or only for create_glossary?